### DISCUSSION TITLE
van-icon支持什么类型的图标？

### DISCUSSION THREAD

**Discussion by ZXHHYJ** [youzan/vant#12573] [Q&A]
我在使用van-tabbar-item时，尝试使用xions库的图标和一些常规的svg似乎都没办法正常显示，有没有适合vant的图标库推荐

**Comment by inottn** [comment]
好像是支持字体图标吧，如果使用 svg 的话，试试使用 [icon slot](https://vant-ui.github.io/vant/#/en-US/tabbar#custom-icon) 自定义展示逻辑。
